### PR TITLE
Leeds tech hub

### PR DESCRIPTION
**Pull request by mohadian**

### CODE REVIEW

File: meetups.html [hunk 1]
```diff
@@ -6,8 +6,23 @@
 
   <div class="row clearfix">
       <div class="skycom-6 meetup-event">
+          <h3><a href="http://www.meetup.com/Leeds-TechHub/events/225199988/">LeedsTechHub - Digital Analytics for Agile: Continuous Performance Optimisation</a></h3>
+          <span class="published-date">26 October 2015, Leeds</span>
+          <p>
+            <b>Jonny Longden</b> talks about the ways in which Sky are transfering curlture and structure to embed analytics in agile so that performance improvement is both data driven and firmly owned by the agile team.
```
Comment (behaghel) [inline meetups.html]: typo on culture, 2 *r* on transferring and Sky is singular which gives _Sky is transferring culture_
I guess that's what you deserve with a PR at 11:40pm :)

File: meetups.html [hunk 2]
```diff
@@ -40,11 +55,107 @@
...
+
+  <div class="row clearfix">
+      <div class="skycom-6">
           <h3><a href="http://www.meetup.com/Couchbase-London/">Couchbase London Meetup 2014</a></h3>
-          <p><b>Julien Gagnet</b> talks about our use of Couchbase as a user profile store</p>
+          <p><b>Julien Gagnet</b> talks about our use of Couchbase as a user profile store.</p>
       </div>
       <div class="skycom-6 video_wrapper">
-              <iframe width="560" height="315" src="//www.youtube.com/embed/GwE5j5vqaMk" frameborder="0" allowfullscreen></iframe>
+          <iframe width="560" height="315" src="https://www.youtube.com/embed/GwE5j5vqaMk" frameborder="0" allowfullscreen></iframe>
```
Comment (behaghel) [inline meetups.html]: Protocol relative link is a good practice. Was it deleted on purpose?

Comment (mohadian) [inline meetups.html]: It was missing `https:` I have added it

File: meetups.html (hunk 1 above)
Comment (mohadian) [inline meetups.html]: Fixed

File: meetups.html [hunk 3]
```diff
@@ -6,8 +6,26 @@
 
   <div class="row clearfix">
       <div class="skycom-6 meetup-event">
+          <h3><a href="http://www.meetup.com/Leeds-TechHub/events/225199988/">LeedsTechHub - Lean UX, Analytics and Performance</a></h3>
+          <span class="published-date">26 October 2015, Leeds</span>
+          <p>
+            <b>Jeff Gothelf</b>, author of the O'Reilly book “Lean UX: Applying lean principles to improve user experience”, takes a look at what he’s learned in the past 5 years, how his own thinking has evolved and what the future of great team collaboration and product development looks like.
+          </p>
+          <p>
+            <b>Jonny Longden</b> talks about the ways in which Sky are transfering culture and structure to embed analytics in agile so that performance improvement is both data driven and firmly owned by the agile team.
```
Comment (behaghel) [inline meetups.html]: Sky **is** transfe**rr**ing

Comment (mohadian) [inline meetups.html]: Fixed

File: meetups.html (hunk 2 above)
Comment (regularfry) [inline meetups.html]: For info: http://www.paulirish.com/2010/the-protocol-relative-url/

Comment (behaghel) [inline meetups.html]: I stumbled upon that when in doubt about my assertion on good practice here. And in fact it's not anymore. Mostafa did the right thing and I stand corrected. Thank you!

**Pull request merged**